From ebe9edd899ea69949c078f4e4cdd6376ebb48495 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 14 May 2017 01:01:13 -0700
Subject: [PATCH] Revert "OMXNodeInstance: use a lock around OMX::freeNode"

This commit breaks the audio and/or video playback on STE devices.

diff --git a/media/libstagefright/omx/OMXNodeInstance.cpp b/media/libstagefright/omx/OMXNodeInstance.cpp
index c1abed6..042764d 100644
--- a/media/libstagefright/omx/OMXNodeInstance.cpp
+++ b/media/libstagefright/omx/OMXNodeInstance.cpp
@@ -498,8 +498,6 @@
             break;
     }
 
-    Mutex::Autolock _l(mLock);
-
     if (mActiveBuffers.size() > 0) {
         freeActiveBuffers();
     }
